### PR TITLE
Display product image on product page

### LINKED ISSUE #115: Add product image to product page

### PR DESCRIPTION
**Pull request by ajayyy**
Closes #115

Desktop

Mobile

### CODE REVIEW

File: server/app/components/product-header.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    name: string;
+  };
+}
+
+const ProductHeader = ({ product }: ProductHeaderProps) => (
+  <>
+    {product?.image_url && (
+      <Card className="mx-auto grid w-full grid-cols-1 gap-1 space-y-4 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
+        <span className="grid justify-center">
+          <img
```
Comment (ricetech) [inline server/app/components/product-header.tsx:15]: Image isn't centered vertically
```suggestion
          <img
            className="my-auto"
```

File: server/app/components/product-header.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            src={product.image_url.replace("US120", "US240")}
+            alt="Product"
+            width={120}
+          />
+        </span>
+
+        <span className="md:col-span-6">
+          <h1 className="text-center text-xl font-semibold md:text-left">
+            {product?.name}
+          </h1>
```
Comment (ricetech) [inline server/app/components/product-header.tsx:25]: `gap` and `space` do the same thing. I'm assuming you were trying to use it to center the text?
```suggestion
      <Card className="mx-auto grid w-full grid-cols-1 gap-1 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
        <span className="grid justify-center">
          <img
            src={product.image_url.replace("US120", "US240")}
            alt="Product"
            width={120}
          />
        </span>

        <span className="md:col-span-6">
          <h1 className="my-auto text-center text-xl font-semibold md:text-left">
            {product?.name}
          </h1>
```

**Review by ricetech — CHANGES_REQUESTED** [review]
What is the purpose of `md:h-full`?

File: server/app/components/product-header.tsx (hunk 2 above)
Comment (ajayyy) [reply server/app/components/product-header.tsx:25]: Text gets misaligned with this

Comment (ricetech) [reply server/app/components/product-header.tsx]: Did you remove `space-y-4` as well?

Comment (ajayyy) [reply server/app/components/product-header.tsx]: image

yea

File: server/app/components/product-header.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  product: {
+    image_url: string;
+    name: string;
+  };
+}
+
+const ProductHeader = ({ product }: ProductHeaderProps) => (
+  <>
+    {product?.image_url && (
+      <Card className="mx-auto grid w-full grid-cols-1 gap-1 space-y-4 md:h-full md:grid-cols-7 md:gap-6 lg:w-3/4">
```
Comment (ricetech) [inline server/app/components/product-header.tsx:13]: What is the purpose of `md:h-full`?

File: server/app/components/product-header.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <span className="grid justify-center">
+          <img
+            className="my-auto"
+            src={product.image_url.replace("US120", "US240")}
```
Comment (ricetech) [inline server/app/components/product-header.tsx:17]: Is there a benefit to using US240 if the width is fixed to 120px anyway?

**Review by ricetech — CHANGES_REQUESTED** [review]

File: server/app/components/product-header.tsx (hunk 2 above)
Comment (ricetech) [reply server/app/components/product-header.tsx]: Try 
```
        <span className="grid justify-center md:col-span-6">
          <h1 className="my-auto text-center text-xl font-semibold md:text-left">
          ```

File: server/app/components/product-header.tsx (hunk 4 above)
Comment (ajayyy) [reply server/app/components/product-header.tsx:17]: yea, it looks better. browser pixels aren't real pixels

File: server/app/components/product-header.tsx (hunk 3 above)
Comment (ajayyy) [reply server/app/components/product-header.tsx]: Removed

**Review by ricetech — APPROVED** [review]

**Pull request merged**